src/datasets/info.py

```diff
@@ -189,13 +206,17 @@ def unique(values):
         features = None
         supervised_keys = None
 
+        # Can be prepared for a pipeline if all members can also be prepared for this specific pipeline
+        task_templates = None  # TODO(sbrandeis)
```

Comment:
Of course 😄 

For context, this method is called when concatenating datasets with the [`concatenate_datasets`](https://github.com/huggingface/datasets/blob/master/src/datasets/arrow_dataset.py#L3027) method. The `DatasetInfo` of the concatenated `Dataset` object is constructed using `DatasetInfo.from_merge`: https://github.com/huggingface/datasets/blob/master/src/datasets/arrow_dataset.py#L3098

What I meant here is that it would be nice to propagate the `task_templates` from the components `Dataset` objects to the resulting concatenated `Dataset` object (I'm happy to reformulate if it's not clear 😅 ). "members" refer to the components `Dataset` objects (the ones being concatenated).

If all the component `Dataset`s can be prepared for the `question_answering` task / pipeline, then the resulting concatenated `Dataset` can also be prepared for the `question_answering` task.

Let me know if this helps 😄